### PR TITLE
Reverse switch statement to avoid needing to change for each new vers…

### PR DESCRIPTION
**Pull request by nift4**
…ion (and modernize build system)

### CODE REVIEW

**Comment by nift4** [comment]
Unless there are unexpected surprises, this should be enough to support Android 15 (and maybe all others after that, too).

**Comment by nift4** [comment]
Test APK: https://github.com/nift4/FakeGApps/releases/download/pr1/app-release.apk

File: settings.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    }
+}
+
+dependencyResolutionManagement {
+    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+    repositories {
+        google()
+        maven("https://api.xposed.info")
+        mavenCentral()
+        maven("https://jitpack.io")
```
Comment (novakovicdavid) [inline settings.gradle.kts:17]: Do we need https://jitpack.io? Seems https://api.xposed.info contains the packages we need.

**Review by novakovicdavid — COMMENTED** [review]
Hi @nift4, thank you for the merge request! :heart:  It looks good, just have a question about the need for jitpack io.

File: settings.gradle.kts (hunk 1 above)
Comment (nift4) [reply settings.gradle.kts]: You're right, I have removed it :)

File: build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+plugins {
+    id("com.android.application") version "8.6.0-rc01" apply false
```
Comment (novakovicdavid) [inline build.gradle.kts:3]: I tried building with an older build of Android Studio, but it complained that 8.6.0-rc01 is too new for it. It seems however that 8.6.0 is already out, and it builds fine with that as well.

**Comment by novakovicdavid** [comment]
Can build & confirm it still works fine. It's nice to be able to build the project without jumping through hoops. 

Thank you again for the work. I'll merge it once the above remark is resolved.

File: build.gradle.kts (hunk 2 above)
Comment (nift4) [reply build.gradle.kts]: changed to 8.6

**Pull request merged**